### PR TITLE
Text.Combine: Adding null-handling case

### PR DESCRIPTION
**Pull request by bgribaudo**

### CODE REVIEW

File: query-languages/m/text-combine.md [hunk 1]
```diff
@@ -13,7 +13,9 @@ Text.Combine(<b>texts</b> as list, optional <b>separator</b> as nullable text) a
   
 ## About
 
-Returns the result of combining the list of text values, `texts`, into a single text value. An optional separator used in the final combined text may be specified, `separator`.
+Returns the result of combining the list of text values, `texts`, into a single text value. Any `null` values present in `texts` are ignored.
+
+An optional separator used in the final combined text may be specified, `separator`.
```
Comment (ehrenMSFT) [inline query-languages/m/text-combine.md:18]:
```suggestion
An optional `separator` used in the final combined text may be specified.
```

File: query-languages/m/text-combine.md [hunk 2]
```diff
@@ -31,7 +33,7 @@ Text.Combine({"Seattle", "WA"})
 
 ## Example 2
 
-Combine text values "Seattle" and "WA" separated by a comma and a space, ", ".
+Combine text values "Seattle" and "WA", separated by a comma and a space, ", ".
```
Comment (ehrenMSFT) [inline query-languages/m/text-combine.md:36]:
```suggestion
Combine text values "Seattle" and "WA", separated by a comma and a space.
```

File: query-languages/m/text-combine.md [hunk 3]
```diff
@@ -42,3 +44,17 @@ Text.Combine({"Seattle", "WA"}, ", ")
 **Output**
 
 `"Seattle, WA"`
+
+## Example 3
+
+Combine values "Seattle", `null`, and "WA", separated by a comma and a space, ", ". (The `null` will be ignored.)
```
Comment (ehrenMSFT) [inline query-languages/m/text-combine.md:50]:
```suggestion
Combine the values "Seattle", `null`, and "WA", separated by a comma and a space. (Note that the `null` is ignored.)
```

**Review by ehrenMSFT — CHANGES_REQUESTED** [review]

**Comment by bgribaudo** [comment]
Thanks, @ehrenMSFT! Changes made!

**Review by ehrenMSFT — APPROVED** [review]
Looks good.

**Comment by DougKlopfenstein** [comment]
#sign-off

**Pull request merged**